src/amazon/dsstne/engine/NNNetwork.cpp

```diff
@@ -2319,7 +2319,7 @@ bool NNNetwork::Validate()
     const NNFloat alpha  = (NNFloat)1.0;
     const NNFloat lambda = (NNFloat)0.0; // regularization parameter (no need for bias test)
     const NNFloat mu     = (NNFloat)0.0; // no momentum
-    const NNFloat epsilon = delta * 10.f;
+    const NNFloat epsilon = delta * 20.f;
```

Comment:
The reason I have suggested including a comment is to ensure that someone else working on this code in the future doesn't make the same change without being aware of the potential issues. The PR description helps here, but probably won't be discovered by anyone working on the code.